spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe "the ingredients index" do
+  before(:each) do
+    @ingredient_1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+    @ingredient_2 = Ingredient.create!(name: "Salt", cost: 4)
+  end
+
+  it "lists all ingredients and their attributes" do
+    visit "/ingredients"
+
+    expect(page).to have_content(@ingredient_1.name)
```

Comment:
This looks good, I would just write it the way that you would expect it to display in the view:
```ruby 
expect(page).to have_content("#{@ingredient_1.name}: #{@ingredient_1.cost}")
```